src/commands/config/__tests__/create.test.ts

```diff
@@ -0,0 +1,81 @@
+import fs from "fs";
+import { prompt } from "inquirer";
+import tmp from "tmp";
+import { deployDocumentStore } from "../../../implementations/deploy/document-store/document-store";
+import { deployTokenRegistry } from "../../../implementations/deploy/token-registry/token-registry";
+import { handler as createTempDNS } from "../../dns/txt-record/create";
+import { CreateConfigCommand } from "../config.type";
+import { handler as createConfig } from "../create";
+import configTemplatePathConfigFile from "./config-template-path.json";
```

Comment:
rename to something like `expected-generated-config-file-with-custom-config.json`

Same for the other

Reply:
done.